### PR TITLE
Triton Power Management Changes

### PR DESCRIPTION
**Pull request by PremchandJujjuri**
->This tool automatically powers off the servers that exceed predefined power limits. 
->This tool generates comprehensive reports detailing each server's vendor, power consumption, and power state. Notably, servers that exceed power limits are highlighted in red.

### CODE REVIEW

**Review by sathishpos — COMMENTED** [review]
No comments. As this was reviewed in person already.

File: config/config_redfish.json [hunk 1]
```diff
@@ -4,44 +4,76 @@
...
+  "logdir": "logdir",
+  "response_codes": {
+    "success": [
+      200,
+      201,
+      202,
+      204
+    ],
+    "failed": [
+      301,
```
Comment (badari-joshi) [inline config/config_redfish.json:30]: Anything which is not in pass block can be a failure. If you want to keep the list of failure codes then it makes sense to have associated failure string as well to the codes.

**Review by deeptis13 — APPROVED** [review]
Reviewed offline and review comments are incorporated in this PR.  No further comments

File: lib/generate_report.py [hunk 2]
```diff
@@ -145,7 +117,7 @@ def generate_hardware_report(self, html_results):
             <table>
               <tr>
                 <th>
-                  <h2>##### System Details #####</h2>
+                  <h2> 9th Floor *** Storage Solutions Lab *** Servers Power Report </h2>
```
Comment (badari-joshi) [inline lib/generate_report.py]: Remove the floor details

**Review by badari-joshi — APPROVED** [review]

File: lib/generate_report.py (hunk 2 above)
Comment (PremchandJujjuri) [reply lib/generate_report.py]: Addressed review comment

File: config/config_redfish.json (hunk 1 above)
Comment (PremchandJujjuri) [reply config/config_redfish.json:30]: Thanks @badari-joshi for the review comment.
Will take this enhancement in next patch

**Review by sathishpos — APPROVED** [review]

**Pull request merged**